src/dataExplorer/shared/utils.ts

```diff
@@ -129,7 +129,7 @@ export const rangeToParam = (timeRange: TimeRange) => {
   }
 }
 
-export const sanitizeSQLSearchTerm = (searchTerm: string): string => {
+export const sanitizeSQLSearchTerm = (searchTerm: string = ''): string => {
```

Comment:
This is a small fix I found when I was refactoring, so just including it in this PR